go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/hashicorp/terraform-plugin-docs v0.14.1
 	github.com/hashicorp/terraform-plugin-framework v1.5.0
 	github.com/hashicorp/terraform-plugin-testing v1.6.0
-	github.com/vantage-sh/vantage-go v0.0.13
+	github.com/vantage-sh/vantage-go v0.0.14-0.20240408182501-72e700aa1c5c
```

Comment:
pseudo release via `go get github.com/vantage-sh/vantage-go@vntg-api-swagger-upgrade` to be replaced by actual release before merge.